Sources/NIO/ChannelPipeline.swift

```diff
@@ -1555,6 +1555,20 @@ extension ChannelPipeline: CustomDebugStringConvertible {
         return desc.joined(separator: "\n")
     }
     
+    /// Returns the first `ChannelHandler` of the given type.
+    ///
+    /// - parameters:
+    ///     - handlerType: the type of `ChannelHandler` to return.
+    public func handler<Handler: ChannelHandler>(type _: Handler.Type) -> EventLoopFuture<Handler> {
+        return context(handlerType: Handler.self).map { context in
```

Comment:
Nit: can we use the explicit `self` keyword here?

Reply:
@Lukasa done!